src/main/java/com/example/doodle_clone/controller/MainController.java

```diff
@@ -1,72 +1,71 @@
 package com.example.doodle_clone.controller;
+
 import com.example.doodle_clone.models.Meeting;
 import com.example.doodle_clone.models.Slot;
-import com.example.doodle_clone.repo.Repository;
+import com.example.doodle_clone.repo.MeetingRepository;
 import com.example.doodle_clone.repo.SlotRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.PathVariable;
+
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 @Controller
 public class MainController {
 
     @Autowired
-    private Repository repository; //репозиторій зустрічі
+    private MeetingRepository meetingRepository; //репозиторій зустрічі
     @Autowired
     private SlotRepository slotRepository;
 
 
-
     @GetMapping("/main_title")
-    public String mainTitle(Model model){
-        Iterable<Meeting> meetings = repository.findAll();
-        model.addAttribute("meetings",meetings);
+    public String mainTitle(Model model) {
+        Iterable<Meeting> meetings = meetingRepository.findAll();
+        model.addAttribute("meetings", meetings);
         return "mainMenu";
     }
 
     @GetMapping("/newTable")
-    public String newTable(Model model){
+    public String newTable(Model model) {
         return "newTable";
     }
 
 
-
     @PostMapping("/newTable")
-    public String createNewTable(@RequestParam String title,@RequestParam String text,@RequestParam String location,@RequestParam int count,
+    public String createNewTable(@RequestParam String title, @RequestParam String text, @RequestParam String location, @RequestParam int count,
```

Comment:
I think I will change to RequesBody when I connect and transfer the JS code to React, so far I have some problems with this